typespec/Applications.Core/environments.tsp

```diff
@@ -79,7 +79,7 @@ model RecipeConfigProperties {
   @doc("Configuration for Terraform Recipes. Controls how Terraform plans and applies templates as part of Recipe deployment.")
   terraform?: TerraformConfigProperties;
 
-  @doc("Environment variables injected during Terraform Recipe execution for the recipes in the environment.")
+  @doc("Environment variables injected during recipe execution for the recipes in the environment.")
```

Comment:
@lakshmimsft what's causing this to change?

Reply:
Correcting the comment to indicate environment variables schema is part of RecipeConfigProperties and is not intended to be specific to Terraform execution.

Reply:
@lakshmimsft it is specific to Terraform though at this point. Unless we are actually injecting this env vars for non-Terraform recipes, we should indicate it is. As a user, I'd then set it assuming it works for bicep as well, which it doesn't.

Reply:
This was in response to a question asked in a design meeting: https://github.com/radius-project/design-notes/pull/53#discussion_r1661395462
Perhaps the comment should be more clear, eg: Environment variables are injected during the execution of recipes within the environment, currently supported for Terraform recipes.